authorization/index.md

```diff
@@ -124,11 +124,20 @@ On receiving the launch notification, the app would query the issuer's
     Accept: application/json
 
 The metadata response contains (among other details) the EHR's authorization
-URL for use below. For details about how the EHR publishes the relevant OAuth URLs, <a href="{{site.baseurl}}authorization/conformance-statement">see here</a>.
+URL for use below. For details about how the EHR publishes the relevant OAuth URLs, <a href="{{site.baseurl}}authorization/conformance-statement">see here</a>.  
+Later, when the app prepares a list of access scopes to request from the EHR authorization server, it will bind to the existing EHR context by including the launch notification in the scope.
+
+#### Standalone launch sequence
+If the app is a standalone app that launches from outside the EHR, it will launch from its registered URL without a launch id.  Such an app will declare its launch context requirements by adding specific scopes to the authorization request.  The EHR’s "authorize" endpoint will acquire the context the app needs and making it available.   
```

Comment:
Got it.

-Dixie


Dixie B. Baker, Ph.D.
Senior Partner
Martin, Blanck and Associates
Office (Redondo Beach, CA):  310-791-9671
Mobile:  310-279-2579

On Mar 20, 2015, at 4:49 PM, Josh Mandel <notifications@github.com<mailto:notifications@github.com>> wrote:


In authorization/index.md<https://github.com/smart-on-fhir/smart-on-fhir.github.io/pull/26#discussion_r26885115>:

> @@ -124,11 +124,20 @@ On receiving the launch notification, the app would query the issuer's
>      Accept: application/json
>
>  The metadata response contains (among other details) the EHR's authorization
> -URL for use below. For details about how the EHR publishes the relevant OAuth URLs, <a href="{{site.baseurl}}authorization/conformance-statement">see here</a>.
> +URL for use below. For details about how the EHR publishes the relevant OAuth URLs, <a href="{{site.baseurl}}authorization/conformance-statement">see here</a>.
> +Later, when the app prepares a list of access scopes to request from the EHR authorization server, it will bind to the existing EHR context by including the launch notification in the scope.
> +
> +#### Standalone launch sequence
> +If the app is a standalone app that launches from outside the EHR, it will launch from its registered URL without a launch id.  Such an app will declare its launch context requirements by adding specific scopes to the authorization request.  The EHR’s "authorize" endpoint will acquire the context the app needs and making it available.


If you could wrap these paragraphs to 80-character lines, that will help us maintain consistent formatting.

—
Reply to this email directly or view it on GitHub<https://github.com/smart-on-fhir/smart-on-fhir.github.io/pull/26/files#r26885115>.